cli/tests/device-appsUpdateTestAll.sh

```diff
@@ -8,9 +8,8 @@ cd `mktemp -d`
 git clone $GIT_REMOTE
 cd ledger-live-common
 yarn
-yalc publish
 cd cli
-yalc add @ledgerhq/live-common
+yarn add @ledgerhq/live-common
```

Comment:
this makes no sense and could have incompatibility issues. we should just yarn. so probably remove the line.